dashboard/app/helpers/levels_helper.rb

```diff
@@ -260,7 +260,7 @@ def app_options
     end
 
     if @script
-      view_options(script_name: @script.name)
+      view_options(script_name: @script.name, script_year: @script.get_course_version.key)
```

Comment:
Some of our course versions have a key "unversioned" which means that it was a one off unit (for example, for hour of code). It looks a little odd in the footer so maybe it's worth not displaying it if the key is "unversioned"? Up to you if you want this logic to be on the client or backend.

From http://localhost-studio.code.org:3000/s/dance-2019/lessons/1/levels/8:
![Screen Shot 2022-10-21 at 12 44 08 PM](https://user-images.githubusercontent.com/46464143/197247192-2ec6754f-37b4-4896-8457-b9c70ba8937c.png)

Reply:
Thanks for pointing out the edge case! Are there others we might run in to? I'm trying to decide if we would benefit from explicitly looking for 'unversioned' and not displaying in that case, or if we should disallow any string that isn't comprised of numbers (therefore only allowing 'years').

Reply:
Updated to filter out all keys that are not entirely numbers. Hopefully leaving only years!
![Screenshot from 2022-10-21 11-39-22](https://user-images.githubusercontent.com/2959170/197266860-50879591-55d4-4103-bc3d-19f890443615.png)

Reply:
I love this approach! Seems more future-proof:)